Add cross compile support

Upstream-Status: Submitted [Maintainer directly]

All the user to specify the build variables in the environment.

Signed-off-by: Darren Hart <dvhart@Linux.intel.com>

Index: gnu-efi-3.0/Make.defaults
===================================================================
--- gnu-efi-3.0.orig/Make.defaults
+++ gnu-efi-3.0/Make.defaults
@@ -54,12 +55,12 @@ LDFLAGS	   = -nostdlib
 INSTALL	   = install
 prefix     = /usr/bin/
 
-CC            = $(prefix)gcc
-AS            = $(prefix)as
-LD            = $(prefix)ld
-AR            = $(prefix)ar
-RANLIB        = $(prefix)ranlib
-OBJCOPY       = $(prefix)objcopy
+CC            := $(prefix)gcc
+AS            := $(prefix)as
+LD            := $(prefix)ld
+AR            := $(prefix)ar
+RANLIB        := $(prefix)ranlib
+OBJCOPY       := $(prefix)objcopy
 
 ifeq ($(ARCH),ia64)
   CFLAGS += -mfixed-range=f32-f127
